transform/models/intermediate/imputation/_imputation.yml

```
@@ -36,6 +36,14 @@ models:
         description: The slope of the occupancy regression
       - name: occupancy_intercept
         description: The intercept of the occupancy regression
+      - name: OTHER_STATION_IS_LOCAL
+        description: tagging wheather the other id is local counters
+          or regional counters. If the other id is just upstream or downstream
```

Proper syntax for a multi-line yaml block:

```suggestion
        description: |
          tagging wheather the other id is local counters
          or regional counters. If the other id is just upstream or downstream
```